operator/config/overlays/development/controller_manager_config.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: config.grafana.com/v1
+kind: ProjectConfig
+health:
+  healthProbeBindAddress: :8081
+metrics:
+  bindAddress: :8080
+webhook:
+  port: 9443
+leaderElection:
+  leaderElect: false
+  resourceName: e3716011.grafana.com
+featureFlags:
+  enableLokiStackWebhook: false
+  enableAlertingRuleWebhook: false
+  enableRecordingRuleWebhook: false
```

Comment:
If I am not missing anything, then all the feature-flags now default to `false`, so these should not be needed?

Reply:
I left this as a reminder to make the development overlay ready to use the webhook.